Dockerfile

```diff
@@ -1,81 +1,36 @@
-#     ³                                                                  ³
-#     ³                                                                  ³
-#     º                                                                  º
-#     ÈÍÍÍÄÄÄÄÄÄÄÄÄÄÄÄÄÄ Ä  ÄÄ  Ä  Ä   ú   Ä  Ä  ÄÄ  Ä ÄÄÄÄÄÄÄÄÄÄÄÄÄÄÄÍÍÍ¼
-#
-#     Greets: ...
-#
-#
-#                                 - [ BROAD‘15 ] -
-#                                                          -0x00B1 [05/06/84]
-FROM ubuntu:14.04
-RUN apt-get -y update                                            && \
-    apt-get -y upgrade                                           && \
-    apt-get -y install                                              \
-      cython                                                        \
-      git                                                           \
-      openjdk-7-jdk                                                 \
-      python-h5py                                                   \
-      python-imaging                                                \
-      python-libtiff                                                \
-      python-lxml                                                   \
-      python-matplotlib                                             \
-      python-mysqldb                                                \
-      python-numpy                                                  \
-      python-pandas                                                 \
-      python-pip                                                    \
-      python-scipy                                                  \
-      python-skimage                                                \
-      python-sklearn                                                \
-      python-vigra                                                  \
-      python-wxgtk2.8                                               \
-      python-zmq                                                    \
-      xvfb
+FROM ubuntu:16.04
+# Install CellProfiler
+RUN   apt-get -y update &&                                          \
+      apt-get -y install                                            \
+        build-essential    \
+        cython             \
+        git                \
+        libmysqlclient-dev \
+        libhdf5-dev        \
+        libxml2-dev        \
+        libxslt1-dev       \
+        openjdk-8-jdk      \
+        python-dev         \
+        python-pip         \
+        python-h5py        \
+        python-matplotlib  \
+        python-mysqldb     \
+        python-scipy       \
+        python-numpy       \
+        python-vigra       \
+        python-wxgtk3.0    \
+        python-zmq
 WORKDIR /usr/local/src
 ARG VERSION=stable
 RUN git clone -b $VERSION https://github.com/CellProfiler/CellProfiler.git
 WORKDIR /usr/local/src/CellProfiler
+# Install CellProfiler at a specific version
+RUN git checkout tags/2.2.0
```

Comment:
You're right, I forgot the other PR had been merged in which used versions. I'll update.